subsys/bluetooth/audio/Kconfig.tmap

```diff
@@ -0,0 +1,17 @@
+# Bluetooth Audio - Telephony and Media Audio Profile (TMAP) options
+#
+# Copyright 2023 NXP
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+config BT_TMAP
+	bool "Telephone and Media Audio Profile [EXPERIMENTAL]"
+	select EXPERIMENTAL
+	help
+	  Enabling this will enable TMAP.
```

Comment:
Should we add dependencies for the other profiles here? 
A TMAP device needs to be either a CAP Initiator or a CAP Acceptor, right?